pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -307,6 +306,17 @@ func InitializeAttributes(trainStmt *ir.TrainStmt) error {
 	constructOptimizers(trainStmt)
 	constructLosses(trainStmt)
 	attrValidator := modelAttr.Update(commonAttributes)
+	if len(modelAttr) == 0 {
```

Comment:
The variable name `modelAttr` makes this function hard to read, maybe can call it `expectedModelAttrs`?

Reply:
Fix later.